app/models/authenticator/httpd.rb

```diff
@@ -136,8 +136,10 @@ def user_attrs_from_external_directory_via_dbus(username)
       ATTRS_NEEDED.each_with_object({}) { |attr, hash| hash[attr] = Array(user_attrs[attr]).first }
     end
 
-    def user_attrs_from_external_directory_via_auth_api(username)
-      HttpdAuthApi.new.user_attrs(username, ATTRS_NEEDED)
+    def user_attrs_from_external_directory_via_dbus_api_service(username)
+      require "httpd_dbus_api"
```

Comment:
Remove the require as autoload should load the class.

Reply:
Oh I see, you probably are hitting autoload issues cause of the `DBus` part.  In that case, this line should be `require_dependency` instead.

Reply:
Alternate, you can probably add an Inflection entry for DBus

Reply:
Yep, because of that DBus vs Dbus.